### ISSUE TITLE
libcap2 is required (at least) for 2019.4

### ISSUE THREAD

**Issue by sbergen** [Unity-CI/docker#46]
**Bug description**

Running `webbertakken/unity-request-manual-activation-file@v2.0-alpha-1` on 2019.4 images fails with
```
Retrieving manual activation file for unity version "2019.4.13f1".
/opt/unity/Editor/Unity: error while loading shared libraries: libcap.so.2: cannot open shared object file: No such file or directory
cp: cannot stat 'Unity_v2019.4.13f1.alf': No such file or directory
```

I'm assuming the same issue affects all other Unity actions also.

**How to reproduce**

Run the following GitHub Action
```
- name: Request manual activation file
  uses: webbertakken/unity-request-manual-activation-file@v2.0-alpha-1
  with:
    unityVersion: 2019.4.14f1
```

**Expected behavior**

Job finishes without errors.

**Additional details**

This issue does not reproduce on 2020.1 images.
The GabLeRoux images reference libcap2 in their .Dockerfiles, but this repo does not.

**Comment by webbertakken** [comment]
Accepting PRs

**Comment by GabLeRoux** [comment]
Here's a related issue someone had on the gitlab example project:  
https://gitlab.com/gableroux/unity3d-gitlab-ci-example/-/issues/138#note_448599129

**Comment by sbergen** [comment]
Can confirm as fixed in the 0.7.0 images, thanks!

(Still have other issues to fix, but this part is now working fine in my PR: https://github.com/sbergen/Responsible/pull/14)
